docs/configuration.rst

```diff
@@ -503,14 +503,14 @@ The following environment variables for the tracer are supported:
      version_added:
         v1.13.0:
 
-   DD_CIVISIBILITY_ITR_ENABLED:
+   DD_CIVISIBILITY_ITR_DISABLED:
```

Comment:
Can you confirm if `DD_CIVISIBILITY_ITR_ENABLED` was a beta environment variable (technically not publicly available)? I assume we won't need a release note to announce removing the former `DD_CIVISIBILITY_ITR_ENABLED`

Reply:
ITR is not even in beta yet, technically.